todo.js

```diff
@@ -57,24 +54,46 @@ export default function Todo() {
 const styles = StyleSheet.create({
     external_View: {
         // transparent: 'true'
-        borderWidth: 1,
         padding: 5,
-        borderRadius: 10,
-        
-        
+        borderRadius: 20,
     },
 
     internal_View: {
-        borderRadius: 10,
+        borderRadius: 15,
         paddingRight: 5,
         borderWidth: 1,
-        padding: 5,
+        paddingBottom: 15,
+        paddingLeft: 10,
+        paddingTop: 15,
+        paddingRight: 10,
+        borderColor: "lightgray",
+        
+    },
+
+    input_Box: {
+        borderWidth: 1,
+        borderColor: "lightgray",
+        borderRadius: 10,
+        height: 100,
     },
 
     third_View: {
         flexDirection: 'row',
         justifyContent: 'space-between',
         alignItems: 'center',
+        paddingLeft: 10,
+        paddingRight: 10,
+    },
+
+    header_text: {
+        fontSize: 20,
+        fontWeight: 'bold',
+        paddingBottom: 10,
+    },
+
+    text_style: {
+        color: 'blue',
```

Comment:
create a new file for constants like color blue. so that you don't repeat strings.